device/mellanox/x86_64-nvidia_sn5400-r0/ACS-SN5400/sai.profile

```diff
@@ -0,0 +1,3 @@
+SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/sai_5400.xml
+SAI_DUMP_STORE_PATH=/var/log/mellanox/sdk-dumps
```

Comment:
Once the PR that move to single common SAI profile file will be added, this should be changed to the same method